lib/tasks/index.rake

```diff
@@ -109,7 +109,11 @@ task :preindex => :environment do
 
       content = blob_content[entry.sha]
       expand!(content, tag_files, blob_content, categories)
-      asciidoc = Asciidoctor::Document.new(content, attributes: {'sectanchors' => ''})
+      content.gsub!(/link:technical\/(.*?)\[(.*?)\]/) do |line|
+          x = /link:technical\/(.*?)\[(.*?)\]/.match(line)
+          line = "link:#{x[1]}\[#{x[2]}\]"
+        end
+      asciidoc = Asciidoctor::Document.new(content, attributes: {'sectanchors' => ''}, doctype: 'book')
```

Comment:
What's going on with the `doctype` here?

Reply:
It is an initial addition to quiet the warnings when generating user-manual.txt (which includes several first level titles).

Reply:
Yes, I've been annoyed by those warnings, too. But I'm wondering if there's any downside to generating all of the other manpages with the `book` type. Should we just be special-casing `user-manual`?